File: 프로그래머스/lv2/12949. 행렬의 곱셈/행렬의 곱셈.java
class Solution {
    public int[][] solution(int[][] arr1, int[][] arr2) {
        int[][] answer = new int[arr1.length][arr2[0].length];
		for(int i = 0; i < arr1.length; i++) {
			for(int j = 0; j < arr2[0].length; j++) {
				for(int l = 0; l < arr2.length; l++) {
					answer[i][j] += arr1[i][l]*arr2[l][j];
				}
			}
		}
        return answer;
    }
}

File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.py
def solution(arr1, arr2):
    answer = []
    for i in arr1:
        li_mul = []
        for idxI_atom in range(len(arr2[0])):
            atom = 0
            for j in range(len(arr2)):
                atom += i[j]*arr2[j][idxI_atom]
            li_mul.append(atom)
        answer.append(li_mul)
    return answer